RecoTracker/MeasurementDet/python/Chi2ChargeMeasurementEstimator_cfi.py

```diff
@@ -4,8 +4,9 @@
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
 
 from RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimatorDefault_cfi import Chi2ChargeMeasurementEstimatorDefault
-Chi2ChargeMeasurementEstimator = Chi2ChargeMeasurementEstimatorDefault.clone()
-Chi2ChargeMeasurementEstimator.clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight'))
+Chi2ChargeMeasurementEstimator = Chi2ChargeMeasurementEstimatorDefault.clone(
+    clusterChargeCut = dict(refToPSet_ = cms.string('SiStripClusterChargeCutTight'))
```

Comment:
this can also get dropped ("string")

Reply:
The string needs to stay (no `refToPSet_` member in the original PSet), and I'd suggest to keep the explicit `PSet` there as well.